docs/src/api.md

```diff
@@ -0,0 +1,22 @@
+# AdvancedMH.jl
+
+Documentation for AdvancedMH.jl
+
+```@contents
+```
+
+## Structs
+```@docs
+StaticMH
```

Comment:
The struct is actually MetropolisHastings, StaticMH is no struct (relates to my comment in the recent StaticMH PR).
```suggestion
MetropolisHastings
```